### PR TITLE
MINOR: updated documentation where RocksDBStore was being used as the…

### PR DESCRIPTION
**Pull request by cfeduke**
… sample class for byte[] versus Bytes examples

### CODE REVIEW

**Review by bbejeck — APPROVED** [review]
Good catch. Thanks for the contribution @cfeduke!  Sorry for the delay in the review.  LGTM.

**Comment by bbejeck** [comment]
ping @guozhangwang @mjsax @vvcephei @ableegoldman for a second review.

**Comment by bbejeck** [comment]
retest this please

File: streams/src/main/java/org/apache/kafka/streams/state/internals/MemoryLRUCache.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
  *
  *  * Note that the use of array-typed keys is discouraged because they result in incorrect ordering behavior.
```
Comment (mjsax) [inline streams/src/main/java/org/apache/kafka/streams/state/internals/MemoryLRUCache.java]: Can we fix this, too? Should not be `*  * Note...` but `* Note...`

**Comment by mjsax** [comment]
@cfeduke This PR need to be rebased to resolve merge conflicts. There are also some open comments.

**Comment by mjsax** [comment]
@cfeduke -- just cycling back to this PR -- can you update it to resolve the conflicts and address the review comments?

**Review by andrewchoi5 — APPROVED** [review]

**Comment by guozhangwang** [comment]
I've pushed a commit to resolve conflicts and merging now. Thanks @cfeduke for the patch!

**Pull request merged**